content/1.7/back-office/dashboard/dashboard-view.md

```diff
@@ -0,0 +1,140 @@
+
+## Call to action
+
+The button to save the changes is displayed between the 2 last fields:
+ - **Let's go!**: When pressed, closes the new window, redirects back to the dashboard.
+
+If the credentials input are incorrect, will show the following error: "Error! PrestaShop was unable to log in to Addons Please check your credentials and your Internet connection".
+
+# Dashboard
+
+## Call to action
+
+On the top right of this field there are 2 buttons:
+
+ - **Configure** - redirects to the "Average bank fees per payment method" page.
+ - **Refresh** - refreshes the stats of the shop.
+
+## Fields
+
+From left to right these are the fields displayed:
+
+ - **Sales** - displays the sales number in the default currency Tax excl. Has a tooltip: "Sum of revenue (excl. tax) generated within the range by orders considered validated.".
+ - **Orders** - displays the orders number. Has a tooltip: "Total number of orders received within the date range that are considered validated.".
+ - **Cart Value** - displays the average cart value in default currency Tax excl. Has a tooltip: "Average Cart Value is a metric representing the value of an average order within the date range. It is calculated by dividing Sales by Orders.".
+ - **Visits** - displays the shop visits number. Has a tooltip: "Total number of visits within the date range. A visit is the period of time a user is actively engaged with your website.".
+ - **Conversion Rate** - displays the conversion rate in percentage. Has a tooltip: "Ecommerce Conversion Rate is the percentage of visits that resulted in an validated order.".
+ - **Net Profit** - displays the net profit in the default currency Tax excl. Has a tooltip: "Net profit is a measure of the profitability of a venture after accounting for all Ecommerce costs. You can provide these costs by clicking on the configuration icon right above here.".
+
+Below there is a graph that can represent each field separately, pressing on one of the fields above will result in graph changing the measures of the graph by the field and the statistics of it. Every selected field has different colouurs which the field gets painted, and the line in the graph. For each field there are 2 separate graphs, one for current period, and the other for previous period. 
+
+When the mouse pointer is on the graph, an additional small table appears listing the exact date and the metrics that it's currently showing.
+
+# Prestashop News
+
+This field displays the news from PrestaShop. By default 2 latest news are displayed with the date of release, and option to "read more" redirects to the article.
+
+Also, there is a link at the bottom "Find more news" which will open a new page -> https://www.prestashop.com/en/blog where all the articles are stored.
+
+# Activity overview
+
+## Call to action 
+
+On the top right of this field there are 2 buttons:
+
+ - **Configure** - redirects to the "Average bank fees per payment method" page.
```

Comment:
On my side, I am not redirected on another page, I stay on the dashboard when I click on configure